autobahn/wamp_invocation.hpp

```diff
@@ -195,13 +200,13 @@ class wamp_invocation_impl
      * Reply to the invocation with positional arguments.
      */
     template <typename List>
-    void result(const List& arguments);
+    void result(const List& arguments, bool intermediate = false);
```

Comment:
Will revoke PR, and resubmit as multiple PRs. 

VS project samples can be removed and kept on a fork. But since they are for samples only not the main project no harm in keeping them. 

Sorry about tabs, not a fan of tabs myself. Will fix.

Intermediate is a provision for progressive calls. Will remove and submit as separate PR when complete and tested.

> On Feb 19, 2016, at 09:00, Tobias Oberstein <notifications@github.com> wrote:
> 
> In autobahn/wamp_invocation.hpp:
> 
> > @@ -195,13 +200,13 @@ class wamp_invocation_impl
> >       * Reply to the invocation with positional arguments.
> >       */
> >      template <typename List>
> > -    void result(const List& arguments);
> > +    void result(const List& arguments, bool intermediate = false);
> Yep. This PR combines multiple things (and it's easier to merge focused PRs). And it's unclear to be too what intermediate is for ..
> 
> —
> Reply to this email directly or view it on GitHub.
>